mdata/store_cassandra.go

```diff
@@ -22,19 +23,21 @@ import (
 const Month_sec = 60 * 60 * 24 * 28
 
 const keyspace_schema = `CREATE KEYSPACE IF NOT EXISTS %s WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1}  AND durable_writes = true`
-const table_schema = `CREATE TABLE IF NOT EXISTS %s.metric (
+const table_schema = `CREATE TABLE IF NOT EXISTS %s.%s (
     key ascii,
     ts int,
     data blob,
     PRIMARY KEY (key, ts)
 ) WITH CLUSTERING ORDER BY (ts DESC)
-    AND compaction = {'class': 'org.apache.cassandra.db.compaction.TimeWindowCompactionStrategy', 'compaction_window_unit': 'DAYS', 'compaction_window_size': '1' }
+    AND compaction = {'class': 'org.apache.cassandra.db.compaction.TimeWindowCompactionStrategy', 'compaction_window_unit': 'HOURS', 'compaction_window_size': '%d' }
```

Comment:
updated it. I tested how it looks when the table gets created, and it resolves the class path when it creates the table so they end up looking pretty much the same:

```
cqlsh:metrictank> CREATE TABLE metrictank.metric_5122 (
              ...     key ascii,
              ...     ts int,
              ...     data blob,
              ...     PRIMARY KEY (key, ts)
              ... ) WITH CLUSTERING ORDER BY (ts DESC)
              ...     AND bloom_filter_fp_chance = 0.01
              ...     AND caching = {'keys': 'ALL', 'rows_per_partition': 'NONE'}
              ...     AND comment = ''
              ...     AND compaction = {'class': 'TimeWindowCompactionStrategy', 'compaction_window_size': '26', 'compaction_window_unit': 'HOURS', 'max_threshold': '32', 'min_threshold': '4'}
              ...     AND compression = {'chunk_length_in_kb': '64', 'class': 'LZ4Compressor'}
              ...     AND crc_check_chance = 1.0
              ...     AND dclocal_read_repair_chance = 0.1
              ...     AND default_time_to_live = 0
              ...     AND gc_grace_seconds = 864000
              ...     AND max_index_interval = 2048
              ...     AND memtable_flush_period_in_ms = 0
              ...     AND min_index_interval = 128
              ...     AND read_repair_chance = 0.0
              ...     AND speculative_retry = '99PERCENTILE';
cqlsh:metrictank> describe table metric_5122

CREATE TABLE metrictank.metric_5122 (
    key ascii,
    ts int,
    data blob,
    PRIMARY KEY (key, ts)
) WITH CLUSTERING ORDER BY (ts DESC)
    AND bloom_filter_fp_chance = 0.01
    AND caching = {'keys': 'ALL', 'rows_per_partition': 'NONE'}
    AND comment = ''
    AND compaction = {'class': 'org.apache.cassandra.db.compaction.TimeWindowCompactionStrategy', 'compaction_window_size': '26', 'compaction_window_unit': 'HOURS', 'max_threshold': '32', 'min_threshold': '4'}
    AND compression = {'chunk_length_in_kb': '64', 'class': 'org.apache.cassandra.io.compress.LZ4Compressor'}
    AND crc_check_chance = 1.0
    AND dclocal_read_repair_chance = 0.1
    AND default_time_to_live = 0
    AND gc_grace_seconds = 864000
    AND max_index_interval = 2048
    AND memtable_flush_period_in_ms = 0
    AND min_index_interval = 128
    AND read_repair_chance = 0.0
    AND speculative_retry = '99PERCENTILE';
```